13-cqrs/1-basic/2-after/ShipmentCommand.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * This class is ONLY responsible to mutates the delivery state
+ *
+ * @author Nadeesha Dilruwan <nadeesha@adsdaq.eu>
+ */
+final class ShipmentCommand
+{
+    /**
+     * @var DateTime
+     */
+    private $delivery;
+
+    /**
+     * @var string
+     */
+    private $proofOfDelivery;
+
+    /**
+     * ShipmentCommand constructor.
```

Comment:
why ?

Reply:
its not good to have comments like that, these are called Noisy comments

Reply:
Here may be there, is a better way to load the current state

Reply:
ah you guys are asking me to remove the comment, not the constructor.. 
I added this comments, because there can be people who are not familiar with PHP. ex in java the constructor is define as the class name. so I'll remove this as the functional name itself explains it ;)

Reply:
obviously there are better way to do this. I really want to keep this as simple as possible. 
ok first let me try to explain what am I trying to say from this examples.

in the CQS file (before) there are two separate methods for read and write. that's the traditional way. so beyond this point both read and write use the same path.

in the after files those two functions are moved to two separate files and it will take completely different paths. 

**This is not real CQRS but this simple concept is what CQRS trying to achieve **